utils/kata-manager.sh

```diff
@@ -244,6 +244,7 @@ Options:
                 Find more details on LTS and Active versions of containerd on
                 https://containerd.io/releases/#support-horizon
  -d           : Enable debug for all components.
+ -D           : Install Docker CLI.
```

Comment:
Can you explain here that the version installed is the packaged version, not the latest release from GH.

I think this distinction is important given that we install the latest Kata and containerd releases.

Reply:
Ah - wait. It's *not* the latest _distro packaged_ version, it's the latest _docker project packaged_ version.